plugins/dap/default.nix

```
@@ -44,6 +44,8 @@ in
 
           dapBreakpointRejected = mkSignOption "R" "Sign to indicate breakpoints rejected by the debug adapter.";
         };
+
+        extensionConfigLua = helpers.mkNullOrOption types.lines "Extension configuration for dap. Don't use this directly!";
```

```suggestion
        extensionConfigLua = mkOption {
          type = types.lines;
          description = ''
            Extension configuration for dap. Don't use this directly !
          '';
          default = "";
          internal = true;
        };
```

I think that this would fit better for this option.